internal/worker/api/errors.go

```diff
@@ -148,18 +148,30 @@ func apiErrorFromEchoError(echoError *echo.HTTPError) ServiceErrorCode {
 
 // Convert an echo error into an AOC compliant one so we send a correct json error response
 func HTTPErrorHandler(echoError error, c echo.Context) {
-	doResponse := func(code ServiceErrorCode, c echo.Context, internal error) {
+	doResponse := func(code ServiceErrorCode, c echo.Context) {
 		if !c.Response().Committed {
 			var err error
 			sec := find(code)
 			apiErr := APIError(code, sec, c)
 
+			internalError, internalErrorOk := echoError.(*echo.HTTPError)
+
 			if sec.httpStatus == http.StatusInternalServerError {
-				c.Logger().Errorf("Internal server error. Internal: %s, Code: %s, OperationId: %s",
-					internal, apiErr.Code, apiErr.OperationId)
+				errMsg := fmt.Sprintf("Internal server error. Code: %s, OperationId: %s", apiErr.Code, apiErr.OperationId)
+
+				if internalErrorOk {
+					errMsg += fmt.Sprintf(", InternalError: %v", internalError)
+				}
+
+				c.Logger().Error(errMsg)
 			} else {
-				c.Logger().Infof("%s, Code: %s, OperationId: %s", internal,
-					apiErr.Code, apiErr.OperationId)
+				msg := fmt.Sprintf("%d Code: %s, OperationId: %s", c.Response().Status, apiErr.Code, apiErr.OperationId)
+
+				if internalErrorOk {
+					msg += fmt.Sprintf(", InternalError: %v", internalError)
+				}
+
+				c.Logger().Warn(msg)
```

Comment:
I thought `Warn` makes more sense because these requests are coming from the worker, which is controlled by us, so non-500 errors should be flagged and looked at.